stable/kommander/values.yaml

```diff
@@ -78,6 +78,14 @@ kubeaddons-catalog:
 grafana:
   enabled: true
 
+  hooks:
+    jobName: grafana-home-dashboard
+    image: dwdraju/alpine-curl-jq
+    secretKeyRef: ops-portal-credentials
+    serviceURL: http://kommander-kubeaddons-grafana.kommander
+    homeDashboardUID: efa86fd1d0c121a26444b636a3f509a8
```

Comment:
Answered my own question [here](https://github.com/mesosphere/charts/blob/ac3a752f90d80dfb81fe3fe1909e20214067f631/stable/kommander/templates/grafana/dashboards/k8s-resources-clusters.yaml#L2550) but i'm wondering if we ever update the chart does the uid change or its set in stone forever. If it changes, we should query by name but if not, then this is fine.

Reply:
The UID would only change if a dev went in there and changed that field - but it's the same with the name, they're [both fields set on the same json](https://github.com/mesosphere/charts/blob/ac3a752f90d80dfb81fe3fe1909e20214067f631/stable/kommander/templates/grafana/dashboards/k8s-resources-clusters.yaml#L2549-L2550). Personally it felt cleaner to me to grab a dashboard by the UID that we set on the dashboard (since we are in charge of creating that dashboard and we know it exists) rather than querying for a name/string (in dcos-monitoring, we grab dashboard by UID). The api call itself also just looks cleaner to me `/api/search/?query=Kubernetes+%2F+Compute+Resources+%2F+Cluster` vs `/api/dashboards/uid/{{ .Values.grafana.hooks.homeDashboardUID`

Reply:
I'm ok with querying for the dashboard by its UID. I think the only downside is that you can't tell at a glance which dashboard we're setting. How about a comment here that mentions the name of the dashboard?

Reply:
Comment is a good compromise. We may add more dashboards in the future so this helps a human to identify the dashboard without having to grep through all for the uid

Reply:
at least we plumbed it to the values file, good call. I am also hesitant about this but we can figure this out later.